packages/web-components/src/presence-badge/presence-badge.styles.ts

```diff
@@ -0,0 +1,96 @@
+import { css } from '@microsoft/fast-element';
+
+export const styles = css`
+  :host {
+    align-items: center;
+    box-sizing: border-box;
+    cursor: default;
+    display: inline-flex;
+    flex-direction: row;
+    font-size: 0;
+    line-height: 0;
+    min-height: 6px;
+    min-width: 6px;
+    position: relative;
+    transition: opacity 0.5s linear;
+    --presence-size: 16px;
+    --presence-mask-width: 2px;
+    --presence-mask-color: #eee;
+    --presence-color-available: #92c353;
```

Comment:
With the new design token infrastructure, the updated way to handle all these colors is for the the component to declare custom design tokens like `presenceBadgeAway`, create a palette from each color, and use a common recipe like `contrastSwatch` with a value of 3 or 4.5 : 1 and use that recipe instead of hard-coded or provided colors. The css variable model doesn't support dark mode or personalization well.